---
# Leave the homepage title empty to use the site title
title: ''
summary: ''
date: 2022-10-24
type: landing

design:
  # Default section spacing
  spacing: '6rem'

sections:
  - block: resume-biography-3
    content:
      # Choose a user profile to display (a folder name within `content/authors/`)
      username: me
      text: ''
      # Show a call-to-action button under your biography? (optional)
      button:
        text: Download CV
        url: uploads/resume.pdf
      headings:
        about: ''
        education: ''
        interests: ''
    design:
      # Use the new Gradient Mesh which automatically adapts to the selected theme colors
      background:
        gradient_mesh:
          enable: true

      # Name heading sizing to accommodate long or short names
      name:
        size: md # Options: xs, sm, md, lg (default), xl

      # Avatar customization
      avatar:
        size: medium # Options: small (150px), medium (200px, default), large (320px), xl (400px), xxl (500px)
        shape: circle # Options: circle (default), square, rounded
  - block: markdown
    content:
      title: 'About me'
      subtitle: ''
      text: |-
        I received my Ph.D. in Bioinformatics at Huazhong Agricultural University in Jun 2021. With the generous guidance of Prof. Jerri Bartholomew, that experience sparked my lasting interest in host-parasite interactions and evolutionary genomics. My research background is in multi-omic insights (genomics, transcriptomics, and proteomics) into the early animal genome evolution and genetic basis of their phenotypic adaptation. And I am very interested in developing multimodal pipelines to discover fundamental principles that control phenotype.

        During my PhD, I investigated the evolutionary genomics of the myxozoans, micro-meter sized parasitic cnidarians, and found a new model of parasite evolution â€“ mosaic evolution (BMC Biology, 2022). I developed a customized comprehensive proteomic reference database (CCPRD) pipeline, which has greatly improved the efficiency and accuracy of proteomic research in non-model organisms. I also applied proteomics, algorithm development, and quantitative genetic analysis to demonstrate that nematocysts may be a key determinant of the adaptive success of cnidarians. I also modelled the relationship between the evolutionary mode of cnidarians and palaeo-environmental change and found that the diversification of cnidarians is predominantly uncoupled from palaeoclimate.

        I have authored and co-authored 23 peer-reviewed publications, including works published on Science Advances, BMC Biology, Biology, Journal of Experimental Biology, and Parasites & Vectors (seven papers are first author and three more in preparation).

        My long-term goal is to become an independent academic researcher advancing the use of multi-omics and AI-driven methods in biomedical genomics. While my Ph.D. work focused on parasitic cnidarians, my current research as a postdoctoral fellow in Dr. Rendong Yang's lab at the Feinberg School of Medicine, Northwestern University, centers on the structural variations of cancer genomes and single-cell omics. I develop long-read and deep learning based tools to decode the regulatory complexity of cancer genomes, with the aim of enabling earlier detection, more precise target discovery, and personalized therapies - hoping that these approaches will help bridge the gap between genomic insights and practical tools for cancer diagnosis and treatment.
    design:
      columns: '1'
  - block: collection
    id: papers
    content:
      title: Featured Publications
      filters:
        folders:
          - publications
        featured_only: true
    design:
      view: article-grid
      columns: 2
  - block: collection
    content:
      title: Recent Publications
      text: ''
      filters:
        folders:
          - publications
        exclude_featured: false
    design:
      view: citation
  - block: cta-card
    demo: true # Only display this section in the HugoBlox Kit demo site
    content:
      title: ðŸ‘‰ Build your own academic website like this
      text: |-
        This site is generated by HugoBlox Kit - the FREE, Hugo-based open source website builder trusted by 250,000+ academics like you.

        <a class="github-button" href="https://github.com/HugoBlox/kit" data-color-scheme="no-preference: light; light: light; dark: dark;" data-icon="octicon-star" data-size="large" data-show-count="true" aria-label="Star HugoBlox/kit on GitHub">Star</a>

        Easily build anything with blocks - no-code required!

        From landing pages, second brains, and courses to academic resumÃ©s, conferences, and tech blogs.
      button:
        text: Get Started
        url: https://hugoblox.com/templates/
    design:
      card:
        # Card background color (CSS class)
        css_class: 'bg-primary-300 dark:bg-primary-700'
        css_style: ''
---
